scram-project-build.file

```diff
@@ -207,7 +207,7 @@ fi
 
 %if "%{?saveDeps:set}" == "set"
 mkdir -p %i/etc/dependencies
-perl %i/config/SCRAM/findDependencies.pl -rel %i -arch %cmsplatf -scramroot $SCRAMV1_ROOT
+python %i/config/SCRAM/findDependencies.py -rel %i -arch %cmsplatf -scramroot $SCRAMV1_ROOT
```

Comment:
Is this the only place we use findDependencies.pl? If it is then looks like we are not using findDependencies.pl from cmsdist but we pick it up from CMSSW/config area.

Reply:
This is only one place where we are using findDependencies.pl script, so probably the file should be removed from cmsdist.
I have also found a note here: https://github.com/cms-sw/cmssw-config/blob/master/ChangeLog
![Screenshot_2020-03-10_17-15-22](https://user-images.githubusercontent.com/52897789/76334326-2ecd4180-62f3-11ea-8463-f88f7f1531fd.png)

Reply:
Current PR should be closed?
New PR created instead for cmssw-config with optimized code: https://github.com/cms-sw/cmssw-config/pull/75

Reply:
yes, we can close this PR or update it to only contain the following changes
- drop  findDependencies.pl.file
- use  findDependencies.py